src/Mod/Path/PathScripts/PathDressupPathBoundaryGui.py

```diff
@@ -147,7 +175,10 @@ def setupFromExisting():
             elif PathJobGui.StockFromExistingEdit.IsStock(self.obj):
                 setupFromExisting()
             else:
-                PathLog.error(translate('PathJob', "Unsupported stock object %s") % self.obj.Stock.Label)
+                PathLog.error(
+                    translate("PathJob", "Unsupported stock object %s")
```

Comment:
Not necessarily for this PR, but long-term -- from a translation point of view, it's best if you can separate the string that must be translated from the part that translators shouldn't edit. So in this case I suggest something like 
```translate("PathJob","Unsupported stock object") + " %s" % self.obj.Stock.Label```
or
```translate("PathJob","Unsupported stock object") + f" {self.obj.Stock.Label}"```
This prevents a translator from accidentally removing the "%s", extra spaces, punctuation, etc. from the string.